### DISCUSSION TITLE
asynchronous sdk for python ?

### DISCUSSION THREAD

**Discussion by rraammiinn** [pocketbase/pocketbase#3086] [Q&A]
is it planed ?

**Comment by ganigeorgiev** [comment]
At least until we reach a more stable version, there are no plans for other official SDKs at the moment.

A community maintained python package exists - https://github.com/vaphes/pocketbase, but I haven't tested it and I'm not sure how it works and whether it supports all web APIs.

**Reply by rraammiinn** [reply]
I think that module works synchronous only so it would be useless for me .

by an asynchronous python sdk I can connect pocketbase using fastapi efficiently and use realtime pocketbase features to simulate cloud functions .
